WcaOnRails/app/models/event.rb

```diff
@@ -22,6 +22,10 @@ def name_in(locale)
     I18n.t(id, scope: :events, locale: locale)
   end
 
+  def cellName
+    name
+  end
```

Comment:
Unfortunately, it's not really possible to get rid of this method, because it's still a database column, so activerecord will automatically generate a `#cellName` method that unwary people might call.

One I'd be happy with would be to continue to define a `cellName` method, but have it raise an exception, so nobody calls it. How do you feel about that?